README.md

```diff
@@ -73,6 +73,45 @@ install:
 
 Now, just setting `GOOS=js GOARCH=wasm` will run your tests using `wasmbrowsertest`. For other CI environments, you have to do something similar.
 
+### Can I use this inside Github Action?
+
+Sure.
+
+Add these lines to your `.github/workflows/ci.yml`
+
+PS: adjust the go version you need in go-version section
+
+```
+on: [push, pull_request]
+name: Unit Test
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.17.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Install chrome
+      uses: browser-actions/setup-chrome@latest
+    - name: Run chrome headless
+      run:  google-chrome --no-sandbox --disable-gpu --headless --remote-debugging-address=0.0.0.0 --remote-debugging-port=9222 &
```

Comment:
I don't believe you need to run Google Chrome. The tool will automatically spawn an instance.